.github/workflows/ci.yml

```diff
@@ -635,21 +632,21 @@ jobs:
           if [[ "${{ secrets.DATABRICKS_TOKEN }}" != "" ]]; \
           then
             echo "Secrets to run Delta Databricks product tests were configured in the repo"
-            echo "::set-output name=have_databricks_secrets::true"
+            echo "have_databricks_secrets=true" >> $GITHUB_OUTPUT
           else
             echo "Secrets to run Delta Databricks product tests were not configured in the repo"
-            echo "::set-output name=have_databricks_secrets::false"
+            echo "have_databricks_secrets=false" >> $GITHUB_OUTPUT
           fi
       - name: Check GCP secrets
         id: check-gcp-secrets
         run: |
           if [[ "${{ secrets.GCP_CREDENTIALS_KEY }}" != "" ]]; \
           then
             echo "Secrets to run GCP product tests were configured in the repo"
-            echo "::set-output name=have_gcp_secrets::true"
+            echo "have_gcp_secrets=true" >> $GITHUB_OUTPUT
```

Comment:
Can we have `yes`/`no` instead of `true`/`false` to avoid misunderstandings / future confusions with `boolean` type?

Reply:
In YAML, that's not a good idea, because `no` also gets typecasted into a boolean, whereas `yes` is not. So it's actually worse than using `true` and `false` where both get cast into a boolean if not quoted.

IMO in YAML, explicit quoting is the right thing to do and should be enough.

But here we already only treat this as a string.